### ISSUE TITLE
FillHandle (autofill) allows readonly columns to be overwritten

### ISSUE THREAD

**Issue by capsloq** [glideapps/glide-data-grid#425]
If you pass `fillHandle={true}` you can use the handle to overwrite the data of readonly columns.
This can be fixed by a guard case in the onCellEdited function:
``` 
const onCellEdited = React.useCallback((cell: Item, newValue: EditableGridCell) => {  
    if (newValue.readonly)
        return;
```
       
However it feels counterintuitive.

**Comment by jassmith** [comment]
Good catch will fix ASAP

**Comment by jassmith** [comment]
Fixed in 5.0.0
